lips/lip-7.md

```diff
@@ -0,0 +1,53 @@
+
+After gathering interest, the facilitator must choose members to be part of the TWG, add them to the proposal, and identify their roles (co-facilitator, contributor, reviewer, etc.). The proposal must have representation from a minimum of 3 Association members to build collaboration and as a tool to determine relevance to the entire association, rather than an individual member.
+
+The facilitator is responsible for promoting and explaining the proposed TWG to Libra members and must lead it through the voting process by the Technical Steering Committee (TSC). They are also responsible for recruiting other members to the TWG as needed on an ongoing basis, and must help find a replacement facilitator if they must cease work on the initiative.
+
+Once the TWG proposal is presented to the TSC, a simple majority vote in favor forms the TWG. The TWG can then proceed in its stated work and make decisions as needed in the areas covered by its focus.
+
+## Decision Making in a TWG
+The TWG facilitator owns the agenda, meetings, and organization of the TWG, but is not a final decision maker. They are there to drive and facilitate discussion, and help allocate resources so that recommendations, reports, or draft LIPs can be created. They are also responsible for providing a bi-weekly update to the association on the progress the TWG is making and any associated documentation being produced. Bi-weekly updates should be posted in the Libra portal and distributed via email/slack to all Association members.
+
+If there are material changes to the TWG’s scope, timeline, or expected output, a revised proposal must be submitted to the TSC for re-approval.
+
+Any TWG member can propose any topic or idea within the scope of the TWG up for discussion with the group. These discussions should be documented, and as the group comes to agreement on how to proceed, there should be clear ownership among TWG members for drafting, editing, obtaining feedback, and finalizing the group’s working product. There does not need to be total agreement in the TWG on the final output, but a supermajority (67%+) vote in favor is required before proceeding.
+
+Once the TWGs final output is ready and the group votes to proceed, it must be shared with the TSC for review and edits. After the editing process, the TSC must approve the output before it can be circulated within the Association.
+
+If the output is a LIP, the TWG will submit the draft LIP as a pull-request to the libra/lips folder, entering the official [LIP process](https://lip.libra.org/overview/). At this point the LIP gets a LIP Manager to navigate the process from becoming a Draft LIP to a committed LIP in Github. 
+
+In addition to assisting with the LIP process, a LIP Manager can also dedicate engineering resources to develop an implementation of the LIP if one does not exist already. This may mean: 
```

Comment:
Do we want to call out that a LIP manager can dedicate eng resources?  Doesn't that hold true to anyone (even if not involved in the TWG)?

Reply:
Sorry, but I'm not understanding your question. Can you say it a different way?